demisto_sdk/commands/download/tests/downloader_test.py

```diff
@@ -1381,3 +1361,53 @@ def test_safe_write_unicode_to_non_unicode(
     result = get_file(dest, suffix)
     assert set(result.keys()) == set(fields)
     assert set(result.values()) == {SENTENCE_WITH_UMLAUTS}
+
+
+@pytest.fixture
+def demisto_client_configure(mocker):
+    mocker.patch(
+        "demisto_sdk.commands.upload.uploader.get_demisto_version",
+        return_value=Version("6.8.0"),
+    )
+    mocker.patch(
+        "demisto_sdk.commands.common.content.objects.pack_objects.integration.integration.get_demisto_version",
+        return_value=Version("6.8.0"),
+    )
+    mocker.patch(
+        "demisto_sdk.commands.common.content.objects.pack_objects.script.script.get_demisto_version",
+        return_value=Version("6.8.0"),
+    )
```

Comment:
Is this part of this PR? 🤔

Reply:
You are right.
It is not relevant to the test any more.
I added it first when i tried to mock the client. 
But I'm not calling the client any more.
Removing it.